script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2.txt

```diff
@@ -13,8 +13,8 @@ Akiha and Kohaku-san exchange glances and then stare at me with a concerned look
 "Fine, I won't ask you.
 Kohaku-san, it was loud last night, wasn't it?"
 <Page3>
-"―――Well, I'm not sure.
-It's true that it was windy last night, but the only unusual thing we found during our midnight patrol was that you were asleep on the bed in your uniform."
+"―――Well... I'm not really sure.//「―――ええっと、どうなんでしょうか。
+It's true that it was windy last night, but the only unusual thing we found during our midnight patrol was that you were asleep in bed wearing your uniform."
```

Comment:
I think _on_ the bed is intentional here? In that he collapsed onto it but didn't get under the covers or anything
> ベッドの上で眠って

Reply:
good call. I'm gonna specify that then